templates/partials/reports_content.html

```diff
@@ -53,7 +53,8 @@
           </li>
           <li ng-if="selection.doc.form && selection.doc.content_type === 'xml'" ng-repeat="field in selection.displayFields" class="indent-{{field.depth}}">
             <label translate>{{field.label}}</label>
-            <p>{{field.value}}</p>
+            <p ng-hide="field.file && field.file.content_type.startsWith('image/')">{{field.value}}</p>
+            <img ng-if="field.file && field.file.content_type.startsWith('image/')" src="data:{{field.file.content_type}};base64,{{field.file.data}}" width="100%"/>
```

Comment:
> We do a similar thing with the `resourceIcon` filter

I'll have to check this out - my first thought was to use jQuery, but sounds like there's a more angular approach available